src/main/java/cn/msdnicrosoft/commandbuttons/CommandButtons.java

```diff
@@ -9,43 +9,38 @@
 import net.minecraft.Util;
 import net.minecraft.client.KeyMapping;
 import net.minecraft.client.Minecraft;
-import net.minecraft.client.gui.chat.ClientChatPreview;
 import net.minecraft.client.player.LocalPlayer;
 import net.minecraft.network.chat.Component;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.jetbrains.annotations.NotNull;
 import org.json.simple.JSONObject;
 import org.lwjgl.glfw.GLFW;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 
 public class CommandButtons implements ModInitializer {
     public static final String MOD_ID = "mgbuttons";
     private static ArrayList<JSONObject> masterCommList;
     private static final Minecraft mc = Minecraft.getInstance();
     public static final Logger LOGGER = LogManager.getLogger(MOD_ID);
 
-    public static void send(String text) {
-        for (String i : text.split(" && ")) {
-            if (i.startsWith("/")) {
-                send(i.substring(1), true);
-            } else {
-                send(i, false);
-            }
-        }
+    public static void send(@NotNull String text) {
+        Arrays.stream(text.split("&&"))
```

Comment:
Introduce the suggestions [here](https://github.com/MSDNicrosoft/Command-GUI-Buttons/commit/fe04cee02162613fbee372f7ae918acf924e8d47#commitcomment-83017111).